addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -14,27 +12,27 @@
         <template id="auth_signup.fields" name="Auth Signup/ResetPassword form fields">
 
             <div class="mb-3 field-login">
-                <label for="login">Your Email</label>
-                <input type="text" name="login" t-att-value="login" id="login" class="form-control form-control-sm" autofocus="autofocus"
-                    autocapitalize="off" required="required" t-att-readonly="'readonly' if only_passwords else None"/>
+                <label for="login" class="form-label">Email</label>
+                <input type="text" name="login" t-att-value="login" id="login" class="form-control" autofocus="autofocus"
+                    autocapitalize="off" required="required" t-att-readonly="'readonly' if only_passwords else None" placeholder="name@example.com"/>
             </div>
 
             <div class="mb-3 field-name">
-                <label for="name">Your Name</label>
-                <input type="text" name="name" t-att-value="name" id="name" class="form-control form-control-sm" placeholder="e.g. John Doe"
+                <label for="name" class="form-label">Name</label>
+                <input type="text" name="name" t-att-value="name" id="name" class="form-control" placeholder="John Doe"
                     required="required" t-att-readonly="'readonly' if only_passwords else None"
                     t-att-autofocus="'autofocus' if login and not only_passwords else None" />
             </div>
 
             <div class="mb-3 field-password pt-2">
-                <label for="password">Password</label>
-                <input type="password" name="password" id="password" class="form-control form-control-sm"
-                    required="required" t-att-autofocus="'autofocus' if only_passwords else None"/>
+                <label for="password" class="form-label">Password</label>
+                <input type="password" name="password" id="password" class="form-control"
+                    required="required" t-att-autofocus="'autofocus' if only_passwords else None" placeholder="••••••••••"/>
```

Comment:
Is that a common thing to do? I can't remember having seen that, but ok :+1:

Reply:
Repeating the label, no, of course :wink: I was thinking of not adding anything as placeholder. But ok :+1: